RESOURCES/FEATURE_FLAGS.md

```diff
@@ -46,7 +46,8 @@ These features are **finished** but currently being tested. They are usable, but
 - CONFIRM_DASHBOARD_DIFF
 - DRILL_TO_DETAIL
 - DYNAMIC_PLUGINS
-- ENABLE_SUPERSET_META_DB: [(docs)](https://superset.apache.org/docs/configuration/databases/#querying-across-databases)
+- DATE_FORMAT_IN_EMAIL_SUBJECT: [(docs)](https://superset.apache.org/docs/configuration/databases/#querying-across-databases)
+- ENABLE_SUPERSET_META_DB: [(docs)](https://github.com/apache/superset/pull/31413)
```

Comment:
It looks like this docs link and the one above it are swapped. 

Also, it would nice to add a docs entry rather than linking to the PR. You can run the docs locally and just add some markdown. Docs on how to do that are [here](https://superset.apache.org/docs/contributing/howtos#contributing-to-documentation).

Reply:
thanks for pointing me to the right direction! i have added an entry and updated the link